packages/pwa-kit-create-app/program.json

```diff
@@ -363,7 +363,7 @@
                 "description": "",
                 "templateId": "retail-react-app",
                 "answers": {
-                    "project.extend": true,
+                    "project.extend": false,
```

Comment:
This is to make sure we test against atleast 1 generated project without extensibility.

The preset is private and is only used by unit test and linter workflows. So we're not breaking anything else.

Reply:
Looks like the reason you're not seeing this change take effect, because this branch did not have the bump-up-dev-versions change yet. 

So I brought in the latest changes from `develop` and now seeing other errors..